src/commands.ts

```diff
@@ -835,7 +831,7 @@ export const insertSection =
     }
 
     const section = schema.nodes.section.createAndFill({
-      category: subsection ? 'MPSectionCategory:subsection' : '',
+      category: subsection ? 'subsection' : '',
```

Comment:
may be we could define an enum in the transform no to refer to names and ids of categories by string?

Reply:
Per discussion with Fahid, this is no longer needed, so let's get rid of it.

Reply:
Should the category be an empty string?

Reply:
I think we don't need to specify it at all.